proto/document/v1/document.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package nitric.document.v1;
+
+import "google/protobuf/struct.proto";
+
+// protoc plugin options for code generation
+option go_package = "nitric/v1;v1";
+option java_package = "io.nitric.proto.document.v1";
+option java_multiple_files = true;
+option java_outer_classname = "Documents";
+option php_namespace = "Nitric\\Proto\\Document\\V1";
+option csharp_namespace = "Nitric.Proto.Document.v1";
+
+// Service for storage and retrieval of simple JSON keyValue
+service Document {
+  // Get an existing document
+  rpc Get (DocGetRequest) returns (DocGetResponse);
```

Comment:
Doc -> Document (keeps to convention set the in the README.)

Reply:
done